### ISSUE TITLE
No train/val split in the Kaggle competition?

### ISSUE THREAD

**Issue by Tai-Wang** [lyft/nuscenes-devkit#26]
Hi,

I think there could have been validation set for this dataset (see https://level5.lyft.com/dataset/download-dataset/), but I don't find it in the Kaggle competition. So I'd like to know is there any train/val split like nuscenes? 

Actually I think it will be more convenient and fair to do ablation studies if there is a given train/val split. :) (Or I may need to submit results on the test set again and again to have the system evaluate them...)

**Comment by ternaus** [comment]
At Kaggle:

Public Leaderboard - validation.
Private Leaderboard - test.
